lte/gateway/c/oai/tasks/amf/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+set(NGAP_C_DIR ${PROJECT_BINARY_DIR}/ngap/r16.0)
+set(NAS5G_C_DIR /home/vagrant/magma/lte/gateway/c/oai/tasks/nas5g)
+set(NAS_C_DIR /home/vagrant/magma/lte/gateway/c/oai)
+set(NAS_DIR /home/vagrant/magma/lte/gateway/c/oai/tasks/nas)
+set(AMFAPP_STATE_CPP_PROTOS mme_nas_state nas_state common_types std_3gpp_types)
+list(APPEND PROTO_SRCS "")
+list(APPEND PROTO_HDRS "")
+
+create_proto_dir("feg" FEG_OUT_DIR)
+
+# S6a
+set(S6ASRV_FEG_CPP_PROTOS s6a_proxy)
+set(S6ASRV_FEG_GRPC_PROTOS s6a_proxy)
+generate_cpp_protos("${S6ASRV_FEG_CPP_PROTOS}" "${PROTO_SRCS}"
+  "${PROTO_HDRS}" ${FEG_PROTO_DIR} ${FEG_OUT_DIR})
+generate_grpc_protos("${S6ASRV_FEG_GRPC_PROTOS}" "${PROTO_SRCS}"
+  "${PROTO_HDRS}" ${FEG_PROTO_DIR} ${FEG_OUT_DIR})
+
+set(LTE_OUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/lte/protos/oai")
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+include_directories("${NAS5G_C_DIR}")
+include_directories("${NAS5G_C_DIR}/include")
+include_directories("${NAS5G_C_DIR}/include/ies")
+include_directories("${NAS_C_DIR}/include/nas")
+include_directories(${LTE_OUT_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../orc8r/gateway/c/common/logging)
```

Comment:
is this needed?

Reply:
@chandra-77 do you need all the statements in this cmake file? Can you please confirm?